src/hypothesis/strategies.py

```diff
@@ -756,8 +756,8 @@ def from_regex(regex):
     Both byte-strings and unicode strings are supported, and will generate
     examples of the same type.
 
-    You can use regex flags (such as :const:`python:re.IGNORECASE`,
-    :const:`python:re.DOTALL` or :const:`python:re.UNICODE`) to control
+    You can use regex flags (such as :const:`re.IGNORECASE <python:re.IGNORECASE>`,
+    :const:`re.DOTALL <python:re.DOTALL>` or :const:`re.UNICODE <python:re.UNICODE>` to control
```

Comment:
I think the "python:" part here can probably be dropped altogether (certainly it doesn't seem to be working correctly as it is). I admit sphinx cross-references are still a little bit of a mystery to me though so I might have this wrong.

Reply:
I think it is correct now. "re.DOTALL" appears as the text (with 'const' formatting), and the part in <> links to the python docs. I took this from a link to a function in re which appears nearby.